_posts/2017-10-11-Stairway-To-Health.md

```diff
@@ -0,0 +1,354 @@
+---
+layout: post
+authors: [michael_vervloet, ines_van_stappen, kevin_van_den_abeele]
+title: 'Stairway to Health with IoT ft. MEAN(TS)'
```

Comment:
The title is a bit confusing when seen in upper case (like on the home page). Especially since it breaks right after "ft."  Perhaps use "and" instead of "ft." ?